src/app/pages/wallets/wallet-details/transaction-editor/pages-wallet-transaction.editor.types.ts

```diff
@@ -13,4 +13,5 @@ export interface IWalletTransactionModalFormType {
   description: FormControl<string | null>;
   date: FormControl<Date | null>;
   type: FormControl<WalletTransactionType | null>;
+  category: FormControl<number | null>;
```

Comment:
Please double check and change everywhere types from `number` to `TServerEntityId` for category